.github/workflows/win.yml

```diff
@@ -177,30 +177,13 @@ jobs:
           path: llvm
           key: llvm-libs-${{ env.CI_LLVM_VERSION }}-msvc
 
-      - name: Download LLVM
-        if: steps.cache-llvm-libs.outputs.cache-hit != 'true'
-        run: |
-          iwr https://github.com/llvm/llvm-project/releases/download/llvmorg-${{ env.CI_LLVM_VERSION }}/llvm-${{ env.CI_LLVM_VERSION }}.src.tar.xz -OutFile llvm.tar.xz
-          (Get-FileHash -Algorithm SHA256 .\llvm.tar.xz).hash -eq "D820E63BC3A6F4F833EC69A1EF49A2E81992E90BC23989F98946914B061AB6C7"
-          7z x llvm.tar.xz
-          7z x llvm.tar
-          mv llvm-* llvm-src
-
-      - name: Download LLVM's CMake files
-        if: steps.cache-llvm-libs.outputs.cache-hit != 'true'
-        run: |
-          iwr https://github.com/llvm/llvm-project/releases/download/llvmorg-${{ env.CI_LLVM_VERSION }}/cmake-${{ env.CI_LLVM_VERSION }}.src.tar.xz -OutFile cmake.tar.xz
-          (Get-FileHash -Algorithm SHA256 .\cmake.tar.xz).hash -eq "B6D83C91F12757030D8361DEDC5DD84357B3EDB8DA406B5D0850DF8B6F7798B1"
-          7z x cmake.tar.xz
-          7z x cmake.tar
-          mv cmake-* cmake
-
       - name: Build LLVM
         if: steps.cache-llvm-libs.outputs.cache-hit != 'true'
         run: |
+          git clone --config core.autocrlf=false -b llvmorg-${{ env.CI_LLVM_VERSION }} --depth 1 https://github.com/llvm/llvm-project.git
```

Comment:
**question:** Is there a specific the reason to switch to `git clone` instead of downlading the tarball?
It's more succinct, of course. But it loses the hash validation.
If we just want to reduce the code, we could consider using an action for downloading release artifacts.

Reply:
Their files are named like `.../llvmorg-18.1.0-rc1/llvm-18.1.0rc1.src.tar.xz` (no hyphen before the second `rc1`) so the substitution wouldn't work. Besides I don't see a problem when we are already using Git for all the other libraries now, including LLVM-C.dll

Reply:
Oh yeah, that's great naming 🤦 
No, I'm fine with it, just wanted to understand the reasoning. Thanks 👍